client/src/rpc_client.rs

```diff
@@ -499,22 +499,16 @@ impl RpcClient {
         Ok(())
     }
 
-    pub fn retry_get_balance(
-        &self,
-        pubkey: &Pubkey,
-        retries: usize,
-    ) -> Result<Option<u64>, Box<dyn error::Error>> {
+    pub fn retry_get_balance(&self, pubkey: &Pubkey, retries: usize) -> ClientResult<u64> {
```

Comment:
There didn't seem to be any real use being made of the `Option` return, so I axed it. I think we might be able to get rid of `retry_get_balance` altogether, actually.

Reply:
>  I think we might be able to get rid of `retry_get_balance` altogether, actually.

I was looking at `retry_get_balance` the other day and I couldn't figure out why it still existed.  🚮

Reply:
All right 🔪